eWonConnector-gateway/src/main/java/org/imdc/ewon/comm/CommunicationManger.java

```diff
@@ -74,79 +147,139 @@ public EwonData queryEwon(Integer id) throws Exception {
    }
 
+   /**
+    * Synchronizes data with given last transaction identifier.
+    * If <code>transactionId</code> equals null, continues without.
+    * @param transactionId last transaction identifier
+    * @return resulting Ewons data
+    * @throws Exception if DataMailbox call fails
+    */
    public EwonsData syncData(Long transactionId) throws Exception {
+      // Store start timestamp
       long start = System.currentTimeMillis();
+
+      // Build and perform DataMailbox call
       String body = EwonUtil.httpGet(transactionId == null
             ? buildDMCall(EwonConsts.DM_CALL_SYNCDATA, EwonConsts.DM_PARAM_CREATE_TRANSACTION, null)
             : buildDMCall(EwonConsts.DM_CALL_SYNCDATA, EwonConsts.DM_PARAM_CREATE_TRANSACTION, null,
                   EwonConsts.DM_PARAM_LAST_TID, transactionId.toString()));
+
+      // Log results from DataMailbox call and return deserialized data
       logResults(start, EwonConsts.DM_CALL_SYNCDATA, body);
       return gson.fromJson(body, EwonsData.class);
    }
 
+   /**
+    * Write tag value for given tag name on given device using Talk2M
+    * @param device Ewon device
+    * @param tagName tag name
+    * @param tagValue tag value
+    * @throws Exception if Talk2M call fails
+    */
    public void writeTag(String device, String tagName, String tagValue) throws Exception {
+      // Create list for parameters
       List<String> params = new ArrayList<>();
+
+      // Build base Talk2M call string
       String directory = EwonConsts.T2M_DIR_GET + device + "/" + EwonConsts.T2M_DIR_RCGI;
 
+      // Add tag name to parameters list
       params.add(EwonConsts.T2M_PARAM_TAGNAME1);
       params.add(tagName);
 
+      // Add tag value to parameters list
       params.add(EwonConsts.T2M_PARAM_TAGVALUE1);
       params.add(tagValue);
 
+      // Add Talk2M API key (from authentication information) to parameters list
       params.add(EwonConsts.T2M_M2W_DEVKEY);
       params.add(authInfo.getDevKey());
 
+      // Build and perform Talk2M call
       EwonUtil.httpGet(buildT2MCall(directory, EwonConsts.T2M_CALL_UPDATETAGFORM,
             params.toArray(new String[params.size()])));
    }
 
+   /**
+    * Get live data for given device using Talk2M
+    * @param device device
+    * @return resulting live data
+    * @throws Exception if Talk2M call fails
+    */
    public String getLiveData(String device) throws Exception {
+      // Create list for parameters
       List<String> params = new ArrayList<>();
+
+      // Build base Talk2M call string
       String directory = EwonConsts.T2M_DIR_GET + device + "/" + EwonConsts.T2M_DIR_RCGI;
 
+      // Add Talk2M API key (from authentication information) to parameters list
       params.add(EwonConsts.T2M_M2W_DEVKEY);
       params.add(authInfo.getDevKey());
 
+      // Build and perform Talk2M call
       params.add(EwonConsts.T2M_PARAM_EXPORTBLOCK);
       params.add(EwonConsts.T2M_PARAM_TAGVALUES);
-
```

Comment:
Whitespace change

Reply:
This was intentional. The comment above the two params.add(...) calls pertains to those two calls, and the return EwonUtil... line as well

Reply:
I still think this belongs as its own commit because the function of adding it is to break up blocks of code functionality. The comment not working for that code block is just a byproduct of that space not existing.